Dockerfile

```diff
@@ -30,9 +30,12 @@ ENV DEBIAN_FRONTEND noninteractive
 
 RUN chmod +x /usr/bin/subspace /usr/local/bin/entrypoint.sh
 
-RUN apt-get update \
-    && apt-get install -y iproute2 iptables dnsmasq socat
+RUN apk add --no-cache \
+    iproute2 \
+    iptables \ 
+    dnsmasq \
+    socat 
 
-ENTRYPOINT [ "/usr/local/bin/entrypoint.sh" ]
+ENTRYPOINT [ "sh", "/usr/local/bin/entrypoint.sh" ]
```

Comment:
Do we need to add the `sh`? If this file has a shebang and is executable then this removed the need to add `sh`

Reply:
Good point issue was  `entrypoint.sh`e was pointing to `bin/bash` this is not needed updated the entrypoint script instead.